src/App.js

```diff
@@ -1,15 +1,69 @@
-import React, { useState } from 'react';
+import React, {useState} from 'react';
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+} from 'react-router-dom';
 import './App.css';
+import HomePage from "./pages/HomePage";
+import LoginPage from "./pages/LoginPage";
+import BlogPostPage from "./pages/BlogPostPage";
+import BlogOverviewPage from "./pages/BlogOverviewPage";
+import Header from "./components/Header";
 
 function App() {
-  // We houden in de state bij of iemand is "ingelogd" (simpele versie)
-  const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
-
-  return (
-    <div>
-      Maak hier jouw prachtige blog-applicatie!
-    </div>
-  );
+    // We houden in de state bij of iemand is "ingelogd" (simpele versie)
+    const [isAuthenticated, toggleIsAuthenticated] = useState(false);
+
+    return (
+        <div>
+            <Router>
+                <Header authenticated={isAuthenticated} setAuthenticated={toggleIsAuthenticated}/>
+                <Switch>
+                    <Route exact path="/">
+                        <HomePage/>
+                    </Route>
+                    <Route exact path="/login">
+                        <LoginPage authenticated={isAuthenticated} setAuthenticated={toggleIsAuthenticated}/>
+                    </Route>
+                    <Route exact path="/blogposts">
+                        <BlogOverviewPage/>
+                    </Route>
+                    <Route path="/blogposts/:id">
```

Comment:
bij de routes voor /blogposts en /blogposts/:{id} moet nog iets van authenticatie komen. Als ik niet ingelogd ben, kan ik toch op de pagina's komen door de url in de browser in te typen.